### PR TITLE
Added the Desktop version

### PR DESCRIPTION
**Pull request by coker2019**
Added the desktop version of my portfolio project with all features.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,320 +7,466 @@
...
-        <li><a href=""></a><img src="images/Vector (2).png" alt=""></li>
-        <li><a href=""></a><img src="images/Vector-last.png" alt=""></li>
-
+      <button>
+        <h2 class="connect btn_effect">LET’S CONNECT</h2>
+      </button>
+      <ul class="media-icon ">
+        <li><a href=""><img src="Twitter.png" alt=""></a></li>
+        <li><a href=""><img src="Linkedin icon.png" alt=""></a></li>
+        <li><a href=""><img src="M.png" alt=""></a></li>
```
Comment (GedeonTS) [inline index.html]: - [ ] Kindly make sure this area looks like the Figma design, by making sure the social icons start at the same point with the text and the "let's connect" title has the same font height and paddings as shown in the Figma design.
Your design:
Screenshot from 2023-03-04 23-32-27

Figma:
Screenshot from 2023-03-04 23-34-20

File: index.html [hunk 2]
```diff
@@ -7,320 +7,466 @@
...
-    <img class="saperator" src="images/Separator Bottom.png" alt="">
-    <div class="lang_container">
-      <ul class="lang_container">
-        <li>
-          <h3 class="lang">Skills</h3>
-          <img class="skills" src="images/Union_arrow.png">
-        </li>
-      </ul>
-      <img class="saperator" src="images/Separator Bottom.png" alt="">
+      <div class="about2">
```
Comment (GedeonTS) [inline index.html]: - [ ] For the about section Kindly make sure it is separated from the Work section by using the right background color (Work-section:#E5E5E5;,About:#fff)
- [ ] Kindly make sure to resize your About section's elements to match the Figma design as shown below :point_down: 

[Screencast from 04-03-2023 23:42:17.webm](https://user-images.githubusercontent.com/97834160/222929997-1641b77f-46fc-469f-8257-d616316970e8.webm)
Screenshot from 2023-03-04 23-47-45

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :

Hi @coker2019 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights :tophat:  :+1: 
- No linter errors :+1: 
- Good use of media queries :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (huseyinbiyik) [reply index.html]: As the previous reviewer requested, please fix the background colors. Works section and About Myself section both have light color but slightly different.
Screenshot 2023-03-05 at 1 45 22 AM
Screenshot 2023-03-05 at 1 46 07 AM

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi @coker2019,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
tenor

 ## Highlights
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- .gitignore used to ignore requested files 📁


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- Kindly fix the mobile version issues, work section cards need to be aligned center, there is an overflowing issue, also some button are looks different. Check the GIF below.
Screen Recording 2023-03-05 at 1 47 16 AM

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag m in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @coker2019    👋 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

image


# Highlights
- Working code ✅ 
- Moderate interface ✅ 


## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

__check the comments for details

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**